packages/drivers/routerlicious-driver/src/documentDeltaConnection.ts

```diff
@@ -47,6 +48,7 @@ export class R11sDocumentDeltaConnection extends DocumentDeltaConnection
             tenantId,
             token,  // Token is going to indicate tenant level information, etc...
             versions: protocolVersions,
+            relayUserAgent: [client.details.environment, ` driverVersion:${driverVersion}`].join(";"),
```

Comment:
Just curious: Aren't we already sending client:IClient as a part of the connectMessage:IConnect. Why not just append driverVersion to the existing client.details.environment?

Reply:
Even if we don't want to do that, why do we want to prepend the env details again here? I see that ODSP driver is also doing the same thing, but it is not clear to me why we need to do this.

Reply:
I believe relayUserAgent was specifically designed for this purpose (similar to a browser's User-Agent string the provides environment and versioning data). Also, this is how ODSP does it, and I do not know how/where client.details.environment is originally set, so I'd rather avoid making a mistake by altering it.